packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -124,6 +125,25 @@ function normalizeInstagramReelURL(url: IUrl) {
   };
 }
 
+// Threads
+// - www.threads.net/:handle/post/:id
+function isThreadsURL(url: IUrl) {
+  return (
+    url.host === "threads.net" &&
+    !!url.pathname.match(/^\/[^\/]+\/[^\/]+\/[^\/]+/)
+  );
+}
+
+function normalizeThreadsURL(url: IUrl) {
+  let [username, , id] = without<string>(url.pathname.split("/"), "");
+  return {
+    attributes: {
+      url: `https://threads.net/@${username}/post/${id}`,
+    },
+    Class: ThreadsEmbed,
+  };
+}
```

Comment:
Can you write a test to verify this logic is correct? This looks like it might add additional `@` characters in the path